data/csp/oci/sle15/drivers.yaml

```diff
@@ -0,0 +1,11 @@
+drivers:
+  _namespace_oci:
+    file:
+      - _attribute
+        name: ixgbe
+    file:
```

Comment:
There should be only one line `file:`.

Reply:
Okay, so all drivers are attrs of one file?

Reply:
`file` is a list and as such produces an XML element for each list element, i.e.:
```
<file name="ixgbe"/>
<file name="ixgbevf"/>
```
Also, and sorry for not noticing earlier, it's `_attributes` and the `:` is missing too. `name` needs to be indented one more step as it's a child of `_attributes`. Alternatively, this can also be a flat list  i.e.:
```
    file:
      - ixgbe
      - ixgbevf
      ...
```
That would require `_map_attribute` to be set in `drivers` somewhere, ideally in https://github.com/SUSE-Enceladus/keg-recipes/blob/develop/data/packages_defattr.yaml, analog to the `packages` version. The `_map_attribute` directive tells the XML generator when encountering a list of strings to produce a dict for each element with a `_attributes` key, and add a child element with the given name and the string as its value. This allows to makes those lists more compact and readable.

Reply:
Hey @smarlowucf , any update on this? The issue with the driver attributes is the only thing that's left before this can be merged AFAICS.

Reply:
Pushed a commit. I think maybe I figured it out. :crossed_fingers: